notice/store.go

```diff
@@ -0,0 +1,56 @@
+package notice
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/target/goalert/permission"
+	"github.com/target/goalert/util"
+	"github.com/target/goalert/validation/validate"
+)
+
+// A Store allows identifying notices for various targets.
+type Store struct {
+	findServicesByPolicyID *sql.Stmt
+}
+
+// NewStore creates a new DB and prepares all necessary SQL statements.
+func NewStore(ctx context.Context, db *sql.DB) (*Store, error) {
+	p := &util.Prepare{DB: db, Ctx: ctx}
+	return &Store{
+		findServicesByPolicyID: p.P(`
+			SELECT COUNT(*)
+			FROM services
+			WHERE escalation_policy_id = $1
+		`),
+	}, p.Err
+}
+
+// Sets a notice for a Policy if it is not assigned to any services
```

Comment:
Should start with `FindAllPolicyNotices`

You can use golint to check: `go run golang.org/x/lint/golint ./notice`